pkg/csi_driver/multishare_ops_manager.go

```diff
@@ -50,14 +50,16 @@ type Workflow struct {
 
 // MultishareOpsManager manages the lifecycle of all instance and share operations.
 type MultishareOpsManager struct {
-	sync.Mutex       // Lock to perform thread safe multishare operations.
-	cloud            *cloud.Cloud
-	controllerServer *controllerServer
+	sync.Mutex         // Lock to perform thread safe multishare operations.
+	cloud              *cloud.Cloud
+	controllerServer   *controllerServer
+	msControllerServer *MultishareController
```

Comment:
what is the need to `msControllerServer` here?

Reply:
Check special notes for reviewer :)